rewrite-java/src/main/java/org/openrewrite/java/style/Autodetect.java

```diff
@@ -65,8 +65,7 @@ public static class Detector {
         private final FindLineFormatJavaVisitor findLineFormat = new FindLineFormatJavaVisitor();
 
         public void sample(SourceFile cu) {
-            // only sample Java sources (extending languages need their own auto-detection)
-            if (cu instanceof J.CompilationUnit) {
+            if (cu instanceof JavaSourceFile) {
```

Comment:
@sambsnyd It seems to me that we shouldn't have to change this now. What am I missing?